### PR TITLE
Issue Solved #5

### LINKED ISSUE #5: Create a skeleton component for the code editor
- Create the component where the person will write code.

### PR DESCRIPTION
**Pull request by pratham9770**
This PR has been raised for issue #5 
Integrated the code editor component into the application. The code editor now includes functionalities for displaying code snippets based on language selection from a dropdown menu, which has also been fixed. 

image
image

### CODE REVIEW

**Comment by pratham9770** [comment]
this request is made for issue #5

File: frontend/src/index.js [hunk 1]
```diff
@@ -3,12 +3,16 @@ import ReactDOM from 'react-dom/client';
 import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import { ChakraProvider } from "@chakra-ui/react";
```
Comment (ravinder-chadha) [inline frontend/src/index.js]: Will using a 3rd party UI provider not make the website heavy due to more number of packages at the end?

**Comment by ravinder-chadha** [comment]
Please edit the PR to add code to `dev` branch

File: frontend/src/index.js (hunk 1 above)
Comment (pratham9770) [reply frontend/src/index.js]: yes sir, it does make the initial rendering a bit late, i'll look into alternatives and change it in the next PR?

**Review by ravinder-chadha — DISMISSED** [review]

**Review by ravinder-chadha — APPROVED** [review]
Seems fine to me

**Pull request merged**